tests/OneraM6/mesh_beam.msh~

```diff
@@ -0,0 +1,48 @@
+% Respect the format, mesh and conectivity have to start in line following NPOIN and NELEM
+
```

Comment:
this file is copied

Reply:
Yeah I will also delete al the temporary files from the repository